pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -1023,7 +1064,7 @@ def _do_check(self):
 
         # on_info_bar_clicked requires self._error to be set, so set it to the
         # list of all errors and warnings that storage checking found.
-        self._error = "\n".join(bootloader_errors + self.errors + self.warnings)
+        self._error = "\n".join(self.errors + self.warnings)
 
         return self._error == ""
```

Comment:
Just noticed that the method has the same content as the one for blivet_gui, except for set_warning warning messages. Not sure if there is anything o do about it at this poing though.

Reply:
You are right. I will try to fix it in another PR.